sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com.azure.ai.formrecognizer/ExtractPrebuiltReceiptAsync.java

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
The directory name for all sample Java files is incorrect.
`/src/samples/java/com.azure.ai.formrecognizer/ExtractPrebuiltReceiptAsync.java`